checkmarx/client.py

```diff
@@ -108,24 +107,25 @@ def _auth(self):
 
     @require_auth
     def request(
-            self,
-            method: str,
-            api_version: str = '1.0',
-            print_error: bool = True,
-            *args, **kwargs
+        self,
+        method: str,
+        api_version: str = '1.0',
+        print_error: bool = True,
+        *args,
+        **kwargs,
     ):
         headers = kwargs.pop('headers', {})
         headers['Authorization'] = f'Bearer {self.auth.access_token}'
         if 'Accept' not in headers:
             headers['Accept'] = f'application/json;v={api_version}'
 
-        res = requests.request(method=method, verify=False, headers=headers, *args, **kwargs)
+        res = requests.request(method=method, headers=headers, *args, **kwargs)
 
         if not res.ok:
-            msg = f'{method} request to url {res.url} failed with {res.status_code=} {res.reason=}'
+            msg = f'{method} request to {res.url=} failed with {res.status_code=} {res.reason=}'
             if print_error:
-                print(msg)
-                print(res.text)
+                ci.util.error(f'Request failed {msg=} {res.text=}')
```

Comment:
nit: status-code might also be of interest

Reply:
you are right. I will add it